nginx/assets/dashboards/NGINX-Overview_dashboard.json

```
@@ -72,7 +72,7 @@
         "layout_type": "ordered",
         "widgets": [
           {
-            "id": 2550775938226891,
+            "id": 8,
```

Keep this the same, since I remember there was a graphical issue last time I synced NGINX where `id` with the same values can cause issues.
```suggestion
            "id": 2550775938226891,
```